Geometry/HcalCommonData/interface/HcalDDDSimConstants.h

```diff
@@ -53,6 +53,7 @@ class HcalDDDSimConstants {
   const std::vector<HcalDetId> & getIdHF2QIE() const {return idHF2QIE;}
   double                    getLayer0Wt(const int det, const int phi, 
 					const int zside) const;
+  int                       getFrontLayer(const int det, const int eta) const;
```

Comment:
@ianna it looks like this is a helper function, not a separate version of `getLayerFront()`. But I agree that the naming scheme could be clearer.